oracle/tests/poseidon_tests.rs

```diff
@@ -169,15 +168,15 @@ mod tests {
         assert_poseidon_3_eq!(
             &[
             ],
-            Fp::from_repr(BigInteger256([15449015445789340742, 8188265220607939614, 14255528068821191610, 3263398758375185362]))
+            Fp::from_repr(BigInteger256([12625032309730357895, 3881775963142723428, 1948451027071626224, 400220142328418896]))
         );
 
         // _rand_fields(1);
         assert_poseidon_3_eq!(
             &[
                 Fp::from_repr(BigInteger256([7268460211608788188, 10132480989041334579, 2339874299280274918, 194293202993774285]))
             ],
-            Fp::from_repr(BigInteger256([2558090059455896147, 7307347974437946425, 4536174362957322879, 2630632011732286938]))
+            Fp::from_repr(BigInteger256([13319422177750956895, 2619256142820001370, 15974443115283230879, 4444502174303366803]))
```

Comment:
Why did these values change out of curiosity? Shouldn't the values for 3 wires PLONK be the same?

Reply:
In my previous commit I had accidentally defined `Parameters3` from `oracle::pasta::fp5 ` params! 
 
Near the top of this file...
```
-    use oracle::pasta::fp5 as Parameters3;
+    use oracle::poseidon::PlonkSpongeConstants3;
+    use oracle::pasta::fp_3 as Parameters3;
```

Reply:
ah, got it!